etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/CreateRawBaseTables.java

```diff
@@ -305,10 +304,11 @@ private void checkAndCreateRawView(String dbName, String stageViewName, String d
 
             Connection con = getHiveJDBCConnection(dbName);
             Statement stmt = con.createStatement();
-            ResultSet rs = stmt.executeQuery("SHOW TABLES LIKE '" + stageViewName + "'");
+            String svName = stageViewName;
+            ResultSet rs = stmt.executeQuery("SHOW TABLES LIKE '" + svName + "'");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) "stageViewName" is provided externally to the method and not sanitized before use. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2077)